hip/matrix/csr_kernels.hip.cpp

```diff
@@ -670,6 +670,33 @@ GKO_INSTANTIATE_FOR_EACH_VALUE_AND_INDEX_TYPE(
     GKO_DECLARE_CSR_ADVANCED_SPGEMM_KERNEL);
 
 
+template <typename ValueType, typename IndexType>
+void spgeam(std::shared_ptr<const DefaultExecutor> exec,
+            const matrix::Dense<ValueType> *alpha,
+            const matrix::Csr<ValueType, IndexType> *a,
+            const matrix::Dense<ValueType> *beta,
+            const matrix::Csr<ValueType, IndexType> *b,
+            matrix::Csr<ValueType, IndexType> *c)
+{
+    auto valpha = exec->copy_val_to_host(alpha->get_const_values());
+    auto vbeta = exec->copy_val_to_host(beta->get_const_values());
```

Comment:
do we have any rule or consideration to use the coefficient on device(POINTER_MODE_DEVICE) or host(POINTER_MODE_HOST)? AFAIK, we use the coefficient like pointer_mode_device in spmv. some routines are only for pointer_mode_host

Reply:
You're probably right, I'll load the values on the GPU instead